_rules/presentational-children-no-focusable-content-307n5z.md

```diff
@@ -127,16 +135,79 @@ This element with `menuitemcheckbox` role has a checkbox as a child. Because the
 </ul>
 ```
 
+#### Failed Example 4
+
+This element with the `tab` role contains an `<a>` element.  The `tab` role has [presentational children][].  The `a` element is included in [sequential focus navigation][].  So the element with the `tab` role fails the rule.  (This tablist implementation is non-functional for users.  It's not meant to function - it's only meant to show roles.)
+
+```html
+<ul role="tablist">
+	<li role="tab">
+		<a href="#">Tab 1</a>
+	</li>
+</ul>
+```
+
+#### Failed Example 5
+
+This element with the `img` role contains an `<a>` element.  The `img` role has [presentational children][].  The `<a>` element is included in [sequential focus navigation][].  So the element with the `img` role fails the rule.
+
+```html
+<span role="img" aria-label="some ASCII art">****** This ASCII art ******* <a href="#">contains a link.</a></span>
+```
+
 ### Inapplicable
 
 #### Inapplicable Example 1
 
-This element has a `link` role which does not have [presentational children][].
+None of the roles involved in this semantic table have [presentational children][].  (The roles are `table`, `row`, `cell`, `button`, and `link`, for the `<table>`, `<tr>`, `<td>`, `<button>`, and `<a>` elements, respectively.)  So this rule does not apply.
 
 ```html
-<a href="https://w3.org">W3C Website</a>
+<table>
+		<thead>
+				<tr>
+						<th colspan="2">semantic table header</th>
+				</tr>
+		</thead>
+		<tbody>
+				<tr>
+						<td><button>button in semantic table - no problem</button></td>
+						<td><a href="#">anchor in semantic table - no problem</a></td>
+				</tr>
+		</tbody>
+</table>
 ```
 
+#### Inapplicable Example 2
+
+This `<a>` element has a `link` role, which does not have [presentational children][].  So this element does not fail this rule, because it's inapplicable. 
+
+```html
+<a href="https://w3.org"><button>W3C Website</button></a>
+```
+
+#### Inapplicable Example 3
+
+The applicability of this rule does not include elements with an <i>explicit</i> role of `presentation`.  Only <i>implicit</i>, through roles with [presentational children][]. 
+
+```html
+<ul>
+	<li role="presentation">
+		<a href="#">inapplicable</a>
+	</li>
+</ul>
+```
```

Comment:
Good point.  I read [test-cases.md](https://github.com/act-rules/act-rules.github.io/blob/d703022a0298f1996f980ee204b5304fa9aecaf4/pages/design/test-cases.md?plain=1).  Its guidance of "Consistency With Accessibility Requirements", combined with [Presentational Roles Conflict Resolution](https://www.w3.org/TR/wai-aria-1.2/#conflict_resolution_presentation_none) make it so that I don't know how to create an inapplicable example which explains this aspect of the rule.

I have seen customers wrongly report a false positive for this rule, and they write something like: "This is a false positive because this element does not have role=presentation.  It has role=tab."   Then I reply: "This rule isn't about an explicit role of presentation.  It's about an /implicit/ role of presentation."  So with this inapplicable example, I was trying to help explain that.  

So I tried something different just now in commit 9f81ae4a95da9ae5b412ccb31199917d9cbef491: I removed this inapplicable example and added some equivalent wording in the "background" section.  What do you think?